vnext/Shared/RuntimeOptions.cpp

```diff
@@ -24,6 +25,17 @@ void __cdecl SetRuntimeOptionInt(string &&name, int32_t value) noexcept {
   g_runtimeOptions.insert_or_assign(std::move(name), value);
 }
 
+void __cdecl SetRuntimeOptionString(const char *name, const char *value) noexcept {
```

Comment:
>const char *name [](http://example.com/codeflow?start=36&length=16)

(nit) why not make it string&&

Reply:
As discussed in the design meeting, these APIs should be ABI-safe.

I'll further refactor to make it namespace-less.